pkg/monitortests/network/legacynetworkmonitortests/networking.go

```diff
@@ -249,7 +249,7 @@ func testPodSandboxCreation(events monitorapi.Intervals, clientConfig *rest.Conf
 	// add our successes
 	ret = append(ret, successes...)
 
-	return append(ret)
+	return ret
```

Comment:
is there a reason for this change?

Reply:
Change was suggested by `make verify`, without fix CI job was failing.